packages/react/src/utilities/decorators/withViewport.tsx

```diff
@@ -88,7 +95,7 @@ export function withViewport<TProps extends { viewport?: IViewport }, TState>(
     }
 
     public componentDidMount(): void {
-      const { skipViewportMeasures, disableResizeObserver } = this.props as IWithViewportProps;
+      const { skipViewportMeasures, disableResizeObserver, delayFirstMeasure } = this.props as IWithViewportProps;
```

Comment:
nit: We could order these alphabetically (this is not a problem per se and it is not consistent in our repo today but I would really like us to make an effort in trying to do this, but feel free to ignore 😄)

```suggestion
      const { delayFirstMeasure, disableResizeObserver, skipViewportMeasures} = this.props as IWithViewportProps;
```

Reply:
I 100%+ agree with you.  Unless there is a great reason to group properties (either in type or when de-structuring), I always like to have them alphabetical.